ui/src/components/system/Backup.vue

```diff
@@ -3183,6 +3217,150 @@ export default {
     cleanLastLog() {
       this.currentDataBackup.lastLog = null;
     },
+    openCheckDestination(b){
+      var context = this;
+      
+      context.currentDataBackup = context.initBackupData();
+      context.currentDataBackup.name = b.name;
+      context.currentDataBackup.checkDestinationDescription = null;
+      context.currentDataBackup.checkDestinationResult = null;
+      
+      var method = "exec";
+      var api = "system-backup/validate";
+      var configObj = null;
+      
+      switch(b.props.VFSType){
+	    case "cifs":
+	      configObj = {
+		    SMBShare: b.props.SMBShare,
+			SMBHost: b.props.SMBHost,
+			SMBLogin: b.props.SMBLogin,
+			SMBPassword: b.props.SMBPassword,
+			HostToPing: b.props.SMBHost
+	      }
+		  break;
+		  	
+	    case "smb":
+	      configObj = {
+		    NFSHost: b.props.NFSHost,
+		    NFSShare: b.props.NFSShare,
+		    HostToPing: b.props.NFSHost
+	      }
+		break;
+		  	
+	    case "usb":
+	      configObj = {
+	        USBLabel: b.props.USBLabel,
+	        HostToPing: null
+	      }
+		break;
+		  	
+		case "sftp":
+		  configObj = {
+		    SftpHost: b.props.SftpHost,
+		    SftpPort: b.props.SftpPort,
+		    SftpUser: b.props.SftpUser,
+		    SftpDirectory: b.props.SftpDirectory,
+		    HostToPing: b.props.SftpHost
+		  }
+		break;
+		  
+		case "webdav":
+		  configObj = {
+		    WebDAVLogin: b.props.WebDAVLogin,
+		    WebDAVPassword: b.props.WebDAVPassword,
+		    WebDAVUrl: b.props.WebDAVUrl,
+		    HostToPing: 'nethserver.org' /* checking internet connection */
+		  }
+		break;
+		  
+		case "b2":
+		  configObj = {
+            B2AccountId: b.props.B2AccountId,
+		    B2AccountKey: b.props.B2AccountKey,
+		    B2Bucket: b.props.B2Bucket,
+		    HostToPing: 'nethserver.org' /* checking internet connection */
+		  }
+	    break;
+		  	
+		case "s3":
+          configObj = {
+		    S3AccessKey: b.props.S3AccessKey,
+		    S3Bucket: b.props.S3Bucket,
+		    S3SecretKey: b.props.S3SecretKey,
+		    S3Host: b.props.S3Host,
+		    HostToPing: 'nethserver.org' /* checking internet connection */
+          }
+		break;
+      }
+      
+      configObj.action = b.props.VFSType == "usb" ? "disk-access" : b.props.VFSType + "-credentials";
+      $("#checkDestinationModal").modal("show");
+      
+      context.exec(
+        [api],
+        configObj,
+        null,
+        function(success) {
+          context.currentDataBackup.checkDestinationDescription = context.$i18n.t('backup.destination_is_reachable');
+          context.currentDataBackup.checkDestinationResult = 'success';
+        },
+        function(error) {
+          console.error(error);
+
+          if(configObj.HostToPing != null){
```

Comment:
If I understand correctly. this line means that no check is done for local resources like USB?